pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -372,37 +368,6 @@ func versionInfo() string {
 	return version
 }
 
-// ServiceAccounts returns the list of service accounts for a service.
-// The XDS server incrementally updates the list, by getting the SA from registries.
-// Same list is used to compute CDS response.
-func (s *DiscoveryServer) ServiceAccounts(serviceName string) []string {
```

Comment:
?? is this not used at all?

Reply:
I searched and can not find where it is used.